File: src/webapp/app/Entities/Content.php
<?php

namespace App\Entities;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Content extends Model
{
    use SoftDeletes;
    //
    protected $fillable = ['title', 'description', 'text', 'note'];
}

File: src/api/internal/models/entities/content.go
package entities

import "time"

type Content struct {
	ID          uint64 `gorm:"primaryKey"`
	ClientId    uint64 `gorm:index`
	Title       string
	Description string
	Text        string
	Note        string
	UpdateAt    time.Time
	CreatedAt   time.Time
	DeletedAt   time.Time
}
